code/game/objects/structures/droppod.dm

```diff
@@ -13,14 +10,14 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 	name = "\improper TGMC Zeus orbital drop pod"
 	desc = "A menacing metal hunk of steel that is used by the TGMC for quick tactical redeployment."
 	icon = 'icons/obj/structures/droppod.dmi'
-	icon_state = "singlepod"
+	icon_state = "singlepod_green"
 	density = TRUE
 	anchored = TRUE
 	layer = ABOVE_OBJ_LAYER
 	resistance_flags = XENO_DAMAGEABLE
 	interaction_flags = INTERACT_OBJ_DEFAULT|INTERACT_POWERLOADER_PICKUP_ALLOWED_BYPASS_ANCHOR
-	soft_armor = list(MELEE = 50, BULLET = 70, LASER = 70, ENERGY = 100, BOMB = 70, BIO = 100, FIRE = 0, ACID = 0)
-	max_integrity = 50
+	soft_armor = list(MELEE = 25, BULLET = 80, LASER = 80, ENERGY = 90, BOMB = 70, BIO = 100, FIRE = 0, ACID = 0)
+	max_integrity = 75
```

Comment:
???

Reply:
tougher in hvh, same effective hp for melee.